Dockerfile

```diff
@@ -40,6 +47,9 @@ RUN apk-install git && \
 RUN cp /logstash-1.5.1/vendor/bundle/jruby/1.9/gems/*/vendor/jar-dependencies/runtime-jars/log4j-1.2.17.jar \
        /logstash-1.5.1/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch*/vendor/jar-dependencies/runtime-jars/
 
+RUN apk-install ca-certificates
+ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
```

Comment:
As discussed on Slack, you're right and this shouldn't actually be needed — I removed it. I'll look into whether we can make use of Alpine 3.3's `java-cacerts` package in a bit.

Thanks!